src/middleware/serverMiddlewareLoader.middleware.ts

```diff
@@ -4,35 +4,57 @@ import bodyParser from 'body-parser'
 import { createLoggerMiddleware } from 'ch-logging'
 import ApplicationLogger from 'ch-logging/lib/ApplicationLogger'
 import cookieParser from 'cookie-parser'
-import * as express from 'express'
+import { Application, NextFunction, Request, RequestHandler, Response } from 'express'
 import { INTERNAL_SERVER_ERROR } from 'http-status-codes'
 import { inject } from 'inversify'
 import { provide } from 'inversify-binding-decorators'
 import NunjucksLoader from './nunjucksLoader.middleware'
 
 import { APP_NAME } from 'app/constants/app.const'
+import initContainer from 'app/inversify.config'
+import SessionService from 'app/services/session/session.service'
+import TYPES from 'app/types'
 
 @provide(ServerMiddlewareLoader)
 export default class ServerMiddlewareLoader {
 
   public constructor(
     @inject(NunjucksLoader) private nunjucks: NunjucksLoader,
-    @inject(ApplicationLogger) private logger: ApplicationLogger) {}
+    @inject(ApplicationLogger) private logger: ApplicationLogger,
+    @inject(SessionService) private sessionService: SessionService
+  ) {}
 
-  public loadServerMiddleware(app: express.Application, directory: string): void {
+  public loadServerMiddleware(app: Application, directory: string): void {
     app.use(bodyParser.json())
     app.use(bodyParser.urlencoded({ extended: true }))
     app.use(cookieParser())
     app.use(createLoggerMiddleware(APP_NAME))
 
+    const sessionMiddleware: RequestHandler = initContainer().get(TYPES.SessionMiddleware)
```

Comment:
nope - do not reinit the container. It is already initialised in app.ts. If you need it, you should inject it (yes, middleware can be injected)